.travis/freetype-config

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
it's used by android target, maybe rename file to android-freetype-config ?

Reply:
Good idea.